stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/suggestion/list/SuggestionListController.kt

```diff
@@ -54,21 +105,34 @@ public class SuggestionListController(
         }
     }
 
+    /**
+     * Displays the list of available command suggestions if any exist.
+     */
     public fun showAvailableCommands() {
         currentSuggestions =
             Suggestions.CommandSuggestions(commands).takeIf { commandsEnabled } ?: Suggestions.EmptySuggestions
     }
 
+    /**
+     * Hides the suggestion list and notifies [suggestionListControllerListener]
+     * that the input doesn't contain commands or mentions.
+     */
     public fun hideSuggestionList() {
+        suggestionListControllerListener?.containsCommands(false)
+        suggestionListControllerListener?.containsMentions(false)
         currentSuggestions = Suggestions.EmptySuggestions
     }
 
+    /**
+     * Returns the suggestion list visibility.
+     */
     public fun isSuggestionListVisible(): Boolean {
         return suggestionListUi.isSuggestionListVisible()
     }
 
     private suspend fun renderSuggestions(suggestions: Suggestions) = withContext(DispatcherProvider.Main) {
         suggestionListUi.renderSuggestions(suggestions)
+        suggestionListControllerListener?.onSuggestionListUiVisibilityChanged(suggestionListUi.isSuggestionListVisible())
     }
 
     private fun String.isCommandMessage() = COMMAND_PATTERN.matcher(this).find()
```

Comment:
It kind of is and isn't.

UX-wise, client side, we cannot know that this is a command, because it shouldn't be.

The reason BE parses it is probably if it splits based on "/" and then gets the entire second part as one string. But htis can be treated as a command + another word, as a non-command, or as a two word command.

I think our regex is okay and users can trick it and push it to BE but nothing should happen I think.